### PR TITLE
Updates to Readme 

### PR DESCRIPTION
**Pull request by chrisiacovella**
I updated the readme file to provide clearer instructions regarding duplicating a repo and what should be included in the readme file of a new forcefield

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
+
+ * Source: [Journal reference(s) if parameters are published (include DOI link)]
+
+ * Forcefield file initially created [DATE] by [Creator(s)]
+
+#### Source Notes:
+< Add specific notes about parameters, e.g., if a set of parameters come verbatim from prior work, provide details and a DOI >
+
+#### Additional Notes:
+< Add any other relevant information. Also For consistency, please define conversion factors used. Note that we strongly suggest follow unit conversions defined in openMM, as detailed below. >
```
Comment (ctk3b) [inline README.md]: Typically stylized as "OpenMM"

File: README.md [hunk 2]
```diff
@@ -1,19 +1,47 @@
...
+ * PI is defined as 3.141592653589 for conversion to radians, consistent with openMM.
```
Comment (ctk3b) [inline README.md]: Can probably ditch the phrase "consistent with OpenMM" in these two lines.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,47 @@
...
+
+
+### [Title of your forcefield]
+
+ * Source: [Journal reference(s) if parameters are published (include DOI link)]
+
+ * Forcefield file initially created [DATE] by [Creator(s)]
+
+#### Source Notes:
+< Add specific notes about parameters, e.g., if a set of parameters come verbatim from prior work, provide details and a DOI >
```
Comment (ctk3b) [inline README.md]: "... parameters come__**s**__..."

File: README.md (hunk 1 above)
Comment (ctk3b) [reply README.md]: Also stray capitalized "F" in "Also For consistency.

Also "we strongly suggest **the following**..."

**Comment by chrisiacovella** [comment]
I addressed these typos and such.

**Pull request merged**